components/toast_wrapper/toast_wrapper.jsx

```diff
@@ -84,7 +84,7 @@ class ToastWrapper extends React.PureComponent {
         }
 
         // show unread toast on mount when channel is not at bottom and unread count greater than 0
-        if (typeof showUnreadToast === 'undefined' && props.atBottom !== null) {
+        if (typeof showUnreadToast === 'undefined' && props.atBottom !== null && props.initScrollOffsetFromBottom > 1000) {
```

Comment:
We want toast value to be either `true` or `false`, but for assigning the value to `showUnreadToast` we need to wait for the scroll position to be set i.e wait for view to scroll to position for new message etc hence we set the first value at this condition.

`if (typeof showUnreadToast === 'undefined' && props.atBottom !== null) {`

This is true when a component is mounted and scroll position is set for the first time because `props.atBottom` will be not be null anymore. But if change it to 

`if (typeof showUnreadToast === 'undefined' && props.atBottom !== null && props.initScrollOffsetFromBottom > 1000) {`
Then it will not set the initial value of `showUnreadToast` anymore.

Instead this should be
```
if (typeof showUnreadToast === 'undefined' && props.atBottom !== null) {
    showUnreadToast = unreadCount > 0 && props.initScrollOffsetFromBottom > 1000;
}
```

Reply:
okay, I will change the logic.

Reply:
Chnages Done

Reply:
Thank you, this looks good, Can you add a test case for less than 1000px?

Reply:
yes I have added a new test case for less than 1000px